public/manifest.json

```diff
@@ -1,25 +1,15 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
-}
+	"manifest_version": 2,
```

Comment:
Suggestion : Manifest Version 2 is deprecating soon Better to use V3 version

![image](https://user-images.githubusercontent.com/72279128/169962930-e4c3695f-0371-44a9-a65b-fd620e50e4df.png)